### PR TITLE
New script to provision an OSD on AWS

### PR DESCRIPTION
**Pull request by agullon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 This directory contains the resources required to get up and running for fault injection testing using Litmus. 
 
+## Provisioning an ephemeral environment (OSD)
+
+To run this experiments and tests a separate environment is needed to not clash with other tests when running faults. To provision a new OSD run the script `./script/provision_osd.sh` or build and run the docker image:
```
Comment (obrienrobert) [inline README.md:7]: A couple of small changes:

1. `./script/` -> `./scripts/`

2. To run this experiments and tests -> To run fault testing scenarios, a separate environment is needed to not clash with other tests.

3. To provision a new OSD > The provision a new OSD cluster, run the`./scripts/provision_osd.sh` script, or build and run the docker image:

File: README.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+
+```sh
+docker build -t provision-osd .
```
Comment (obrienrobert) [inline README.md:10]: Should this command be `docker build -t provision-osd scripts/.`, so it references the docker file in the script directory?

File: README.md (hunk 1 above)
Comment (agullon) [reply README.md:7]: will correct it, thanks

File: README.md (hunk 2 above)
Comment (agullon) [reply README.md:10]: yes, you're right, thanks

**Comment by obrienrobert** [comment]
@agullon just a couple of very small typos etc. I tried provisioning a cluster but it seems like there's an issue with OCM at the moment, but looks good to me!

**Comment by obrienrobert** [comment]
@agullon I was able to successfully provision a cluster using the script. 👍🏼

**Comment by agullon** [comment]
I'll merge it then

**Pull request merged**